internal/checks/source-code/code-changes/rules.rego

```diff
@@ -113,6 +108,13 @@ CbPolicy[msg] {
 	msg := {"ids": ["1.1.3", "1.1.4", "1.1.5", "1.1.6", "1.1.9", "1.1.10", "1.1.11", "1.1.12", "1.1.13", "1.1.14", "1.1.15", "1.1.16", "1.1.17"], "status": constsLib.status.Unknown, "details": constsLib.details.repository_missing_minimal_permissions}
 }
 
+#Missing minimal permission for branch protection settings
+CbPolicy[msg] {
+	input.Repository.Collaborators != null
```

Comment:
when can it be null? what will happen in this scenario?

Reply:
It could be null if the token misses repo scope, the next line look for greater check that make 113 redundancy but let's keep it just in case we miss some corner case